examples/recovery/transliterated.json

```diff
@@ -0,0 +1,21 @@
+{
+  "ver": "1.3.0",
+  "nam": {
+    "fn": "ابو بكر محمد بن زكريا الرازي",
+    "fnt": "ABU<BAKR<MOHAMMED<IBN<ZAKARIA<AL<RAZI",
+    "gn": "ناصر",
```

Comment:
Can someone else verify this transliteration? I have checked it via multiple online tools and they all agree. However a second set of eyes / check would be appreciated.

Reply:
`fnt` transliteration seems legit. Glyphs of the `gn` seem to be a bit font-sensitive, with an alternative transliteration of "Naser" suggested.

Reply:
Nasir and Naser are suggested by translation tools, Nasr by transliteration tools.